pkg/cmd/flags/filter.go

```diff
@@ -130,6 +139,7 @@ func prepareEventsToTrace(eventFilter eventFilter, eventsNameToID map[string]eve
 				return nil, InvalidEventExcludeError(name)
 			}
 			isExcluded[id] = true
+			delete(eventsToDisable, name)
```

Comment:
What if the event in "eventsToDisable" is also used in another policy ? You just removed it from the map that is going to be used by the `prepareEventsToTracee` when the other policy filters are parsed, no ? The event could have been excluded in this policy, but maybe not the other ? Am I missing something ?

Reply:
Discussed a bit offline with Geyslan. We agreed (and he already thought like that before) that the signature and the event should be disabled if the ksymbol isnt available, but using a debug log only (so it does not scare the end user).

Reply:
Yep. From the start I didn't like this approach, it increases complexity just to silence an error that should be issued.